src/fsharp/FSharp.Core/Linq.fs

```diff
@@ -87,148 +87,152 @@ module LeafExpressionConverter =
                         if isg1 then minfo2.IsGenericMethod && gmd = minfo2.GetGenericMethodDefinition()
                         else minfo = minfo2
                      ) ->
-                Some (obj, (minfo2.GetGenericArguments() |> Array.toList), args)
```

Comment:
What's this change please?

Reply:
Related to `SpecificCallToMethod` vs `SpecificCallToGenericMethod`.